pkgs/jnigen/test/java_core_generation.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2025, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:jnigen/src/config/config_types.dart';
+import 'package:test/test.dart';
+
+import 'test_util/test_util.dart';
+
+void main() {
+  test('Java core libraries are generated without providing class path',
+      () async {
+    await generateAndAnalyzeBindings(
```

Comment:
This test doesn't seem to check that the generated bindings contain the classes.